open-ce/build_env.py

```diff
@@ -43,31 +43,26 @@
 import test_feedstock
 from errors import OpenCEError, Error
 
-def make_parser():
-    ''' Parser for input arguments '''
-    arguments = [utils.Argument.CONDA_BUILD_CONFIG, utils.Argument.OUTPUT_FOLDER,
-                 utils.Argument.CHANNELS, utils.Argument.ENV_FILE,
-                 utils.Argument.REPOSITORY_FOLDER, utils.Argument.PYTHON_VERSIONS,
-                 utils.Argument.BUILD_TYPES, utils.Argument.MPI_TYPES,
-                 utils.Argument.CUDA_VERSIONS, utils.Argument.SKIP_BUILD_PACKAGES,
-                 utils.Argument.RUN_TESTS, utils.Argument.DOCKER_BUILD]
-
-    parser = utils.make_parser(arguments,
-                               description = 'Build conda environment as part of Open-CE')
-
-    parser.add_argument(
-        '--git_location',
-        type=str,
-        default=utils.DEFAULT_GIT_LOCATION,
-        help='The default location to clone git repositories from.')
-
-    parser.add_argument(
-        '--git_tag_for_env',
-        type=str,
-        default=None,
-        help='Git tag to be checked out for all of the packages in an environment.')
-
-    return parser
+COMMAND = "env"
+
+DESCRIPTION = 'Build conda environment as part of Open-CE'
+
+ARGUMENTS = [utils.Argument.CONDA_BUILD_CONFIG, utils.Argument.OUTPUT_FOLDER,
+             utils.Argument.CHANNELS, utils.Argument.ENV_FILE,
+             utils.Argument.REPOSITORY_FOLDER, utils.Argument.PYTHON_VERSIONS,
+             utils.Argument.BUILD_TYPES, utils.Argument.MPI_TYPES,
+             utils.Argument.CUDA_VERSIONS, utils.Argument.SKIP_BUILD_PACKAGES,
+             utils.Argument.RUN_TESTS, utils.Argument.DOCKER_BUILD,
+             (lambda parser: parser.add_argument(
```

Comment:
Maybe we should just add all of the argument descriptions to the utils file, even if they are only being used in a single place.

Reply:
Yeah, I was considering this.

Reply:
Centralizing the argument descriptions makes sense to me.

Reply:
Done.